gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManagementTaskStreamImpl.java

```diff
@@ -146,7 +147,7 @@ public DagTask next() {
           }
         } catch (Exception e) {
           //TODO: need to handle exceptions gracefully
-          log.error("Exception getting DagAction from the queue / creating DagTask", e);
+          log.error("Exception getting DagAction from the queue / creating DagTask {}", dagAction == null ? "" : dagAction, e);
```

Comment:
wouldn't it just print the string `null`?  seems clearer and more helpful indicator when debugging than is empty string